docs/configuration.rst

```diff
@@ -521,6 +521,15 @@ The following environment variables for the tracer are supported:
      version_added:
         v1.13.0:
 
+   DD_CIVISIBILITY_ITR_ENABLED:
```

Comment:
Is this the right name for this env var? The description makes it seem more of a pre-requisite for ITR rather than a flag for enabling ITR

Reply:
It's a name I got from @juan-fernandez. The description says what it actually does on this branch, and I expect we'll update the description later on when it does more stuff.

Reply:
to give more context: the only thing we use packfiles for is ITR right now. While we could be more granular with feature flags, we don't want to clutter things prematurely. If at some point packfiles are used for something else and we want them regardless of ITR, we can create _another_ feature flag for customers who want to opt out.